specs/rollup-node.md

```diff
@@ -96,6 +152,11 @@ entries.
 
 [deposit-contract-spec]: deposits.md#deposit-contract
 
+Each of the derived `PayloadAttributes` starts with a L1 Attributes transaction.
+
+The [User-deposited] transactions are all put in the first of the derived `PayloadAttributes`,
+inserted after the L1 Attributes transaction, before any [sequenced][g-sequencing] transactions.
+
```

Comment:
The reason is to simplify L2 sync, by making it always possible to read the L1 information from the block-data, instead of reading from some L2 state. The nice thing is that we do not have to batch-submit this part, it's just part of the derivation process, so it only contains us a little bit of L2 disk storage.

Reply:
Added rationale in [f8ccf4a](https://github.com/ethereum-optimism/optimistic-specs/pull/223/commits/f8ccf4a14b89029deefa20a3c004185de56daff4)